test/support/mock_iss/server.js

```diff
@@ -248,6 +248,18 @@ app.get('/api/v3/search/', (req, res) => {
                 services.domesticviolence,
             ],
         });
+    } else if (req.query.q.match(/elasticsearch/)) {
+        res
+            .status(503)
+            .json({
+                error_message: 'The Elasticsearch service is unavailable.',
+            });
+    } else if (req.query.area == "carlt") {
```

Comment:
Should be higher in the branches. Else a valid search will override it, and it's effectively always an error.